src/utility/HCISharedMemTransport.cpp

```diff
@@ -737,14 +864,79 @@ int HCISharedMemTransportClass::bt_ipm_set_power(void)
   data[1] = (uint8_t)(ACI_HAL_SET_TX_POWER_LEVEL & 0x000000FF); /* the OPCODE */
   data[2] = (uint8_t)((ACI_HAL_SET_TX_POWER_LEVEL & 0x0000FF00) >> 8);
   data[3] = 2; /* the length */
-  /* fill the ACI_HAL_WRITE_CONFIG_DATA */
+  /* fill the SET_POWER */
   data[4] = 0x01; /* En_High_Power */
   data[5] = CFG_TX_POWER; /* PA_level */
 
-  /* send the ACI_HAL_WRITE_CONFIG_DATA */
+  /* send the SET_POWER */
   mbox_write(data[0], 5, &data[1]);
   /* now wait for the corresponding Rx event */
   return 1; /* success */
 }
 
+int HCISharedMemTransportClass::bt_ipm_gatt_init(void)
+{
+  /* the specific table for gatt init */
+  uint8_t data[4];
+
+  phase_gatt_init = false;
+
+  data[0] = BT_BUF_CMD; /* the type */
+  data[1] = 0x01; /* the OPCODE */
+  data[2] = 0xFD;
+  data[3] = 0; /* the length */
+
+  /* send the GATT_INIT */
+  mbox_write(data[0], 3, &data[1]);
+  /* now wait for the corresponding Rx event */
+  return 1; /* success */
```

Comment:
`mbox_write` return a value? Maybe it should be fine to check ? And return a good status?

Reply:
I leave Francois to comment because I have only used the same approach used by him on the other messages

Reply:
OK. Missed that when introducing WB...
@FRASTM any comments on this ?

Reply:
I've got a feedback from @LMESTM. This approach is the one used by the cube. Like @FRASTM copy it from the Cube this explain why it is like this. Even if it could probably be more efficient/robust to check the return value. I guess it can be kept like this. If we want harden all the code from the Cube, all sources should be reviewed not only this part.

Reply:
The `mbox_write` return value is the nb of bytes actually sent. If this value is not the _len_ parameter,  then it could be considered as an error. (Raising a new PR for that.)